src/routes/plans/+page.svelte

```diff
@@ -6,8 +6,11 @@
   import { page } from '$app/stores';
   import PlanIcon from '@nasa-jpl/stellar/icons/plan.svg?component';
   import type { ICellRendererParams, ValueGetterParams } from 'ag-grid-community';
+  import XIcon from 'bootstrap-icons/icons/x.svg?component';
   import { flatten } from 'lodash-es';
   import { onDestroy, onMount } from 'svelte';
+  import ExportIcon from '../../assets/export.svg?component';
```

Comment:
Could we mirror the plan form import button design in the plan page export?
<img width="364" alt="image" src="https://github.com/user-attachments/assets/885145e9-7ca4-4d39-a846-578bc4896d46">

<img width="356" alt="image" src="https://github.com/user-attachments/assets/11130d51-764d-451c-a6b6-d44aa56fe3ba">

Reply:
Yup! I already forgot about that spot...